code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -484,19 +484,28 @@
 				if(!(G.is_hidden(src)))
 					. += "<span class='notice'>[t_He] has exposed genitals... <a href='?src=[REF(src)];lookup_info=genitals'>Look closer...</a></span>"
 					break
-	var/line = span_notice("<a href='?src=[REF(src)];lookup_info=open_examine_panel'>Examine closely...</a>")
-	if(line)
-		. += line
+	var/flavor_text_link
+	var/preview_text = client.prefs.read_preference(/datum/preference/text/flavor_preview)
```

Comment:
just get the first 70 characters of your actual flavor text, this doesn't need to be its own pref

Reply:
I did this for a reason, mainly so people could seperate the preview and their flavor text (mainly just because I thought "why not?")

Also, I can't really think of a good way to communicate to the player what the character limit is without adding a default value to a text box, so that's another reason why I did it this way

I might change the way it works to change the preview to an override instead and just have it default to showing the first x amount of characters in flavor text in the absence of anything in the override

Reply:
nevermind it s fine